### PR TITLE
Merge 2020-05-19 into tensorflow

### PR DESCRIPTION
**Pull request by asuhan**

### CODE REVIEW

**Comment by asuhan** [comment]
@swift-ci Please clean test TensorFlow

**Comment by asuhan** [comment]
@swift-ci Please clean test TensorFlow

**Comment by asuhan** [comment]
swift-ci Please clean test TensorFlow.

**Comment by asuhan** [comment]
@swift-ci Please clean test TensorFlow.

**Comment by asuhan** [comment]
@swift-ci Please clean test TensorFlow

**Comment by asuhan** [comment]
@swift-ci Please clean test TensorFlow

**Comment by asuhan** [comment]
@swift-ci Please clean test TensorFlow

**Comment by asuhan** [comment]
@swift-ci Please clean test TensorFlow

**Comment by asuhan** [comment]
@swift-ci Please clean test TensorFlow

**Comment by asuhan** [comment]
@swift-ci Please clean test TensorFlow

**Comment by asuhan** [comment]
@swift-ci Please clean test TensorFlow

**Comment by asuhan** [comment]
@swift-ci Please clean test TensorFlow

**Comment by asuhan** [comment]
@swift-ci Please clean test TensorFlow

File: utils/update_checkout/update-checkout-config.json [hunk 1]
```diff
@@ -351,25 +351,26 @@
...
-                "llvm-project": "a2905f3b8df7784e5c455390c6c0b0f7564e00e4",
+                "llvm-project": "2e4f104624f9cdc7c8cb471a393f496b48922c0b",
                 "swift": "tensorflow",
-                "cmark": "swift-DEVELOPMENT-SNAPSHOT-2020-05-11-a",
-                "llbuild": "swift-DEVELOPMENT-SNAPSHOT-2020-05-11-a",
+                "cmark": "swift-DEVELOPMENT-SNAPSHOT-2020-05-18-a",
+                "llbuild": "swift-DEVELOPMENT-SNAPSHOT-2020-05-18-a",
                 "swift-tools-support-core": "0.0.1",
-                "swiftpm": "swift-DEVELOPMENT-SNAPSHOT-2020-05-11-a",
+                "swiftpm": "swift-DEVELOPMENT-SNAPSHOT-2020-05-20-a",
```
Comment (asuhan) [inline utils/update_checkout/update-checkout-config.json:359]: @ematejska NB: I needed to adjust the swiftpm dependency to pick up https://github.com/apple/swift-package-manager/commit/3f55ea6d555ee8a73013eea44a50567fb69e9f41.

File: utils/update_checkout/update-checkout-config.json [hunk 2]
```diff
@@ -41,7 +41,7 @@
             "platforms": [ "Linux" ]
         },
         "yams": {
-            "remote": { "id": "jpsim/Yams" }
+            "remote": { "id": "asuhan/Yams" }
```
Comment (asuhan) [inline utils/update_checkout/update-checkout-config.json:44]: @ematejska Required until https://github.com/jpsim/Yams/pull/260 (or a better version of it) is merged and released.

**Comment by asuhan** [comment]
@ematejska PTAL, only two snags in utils/update_checkout/update-checkout-config.json, rest of the merge is pristine `tensorflow-stage` from our merge bot.

**Comment by asuhan** [comment]
@swift-ci Please clean test TensorFlow

**Comment by asuhan** [comment]
@swift-ci Please clean test TensorFlow

File: utils/update_checkout/update-checkout-config.json (hunk 2 above)
Comment (ematejska) [reply utils/update_checkout/update-checkout-config.json:44]: Got it.

**Review by ematejska — APPROVED** [review]
Seems like the right tradeoff to me to use the forked yams for now.

File: utils/update_checkout/update-checkout-config.json (hunk 1 above)
Comment (ematejska) [reply utils/update_checkout/update-checkout-config.json:359]: Ok, got it.

**Pull request merged**